### ISSUE TITLE
4 space file won't be flagged when using 2 spaces in editorconfig

### ISSUE THREAD

**Issue by vanniktech** [shyiko/ktlint#187]
Take this file as an example:

```kotlin
package com.test.detekttest

import android.support.v7.app.AppCompatActivity
import android.os.Bundle

class MainActivity : AppCompatActivity() {
    val ab = 999.99

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }
}
```

with the following editorconfig:

```
[*.{kt,kts}]
indent_size=2
```

and ktlint 0.21.0 no styling issues are reported.

**Comment by shyiko** [comment]
https://github.com/shyiko/ktlint/issues/141#issuecomment-361203788 dupe

**Comment by vanniktech** [comment]
Already thought that they might be similar. Should we keep both?
